config/config.go

```diff
@@ -119,7 +119,7 @@ var (
 
 		// Backoff times for retrying a batch of samples on recoverable errors.
 		MinBackoff: model.Duration(30 * time.Millisecond),
-		MaxBackoff: model.Duration(100 * time.Millisecond),
+		MaxBackoff: model.Duration(1000 * time.Millisecond),
```

Comment:
Hmm, the default max backoff change is not really related to handling 429s better. Could that be part of a different PR if we want to do it?

Reply:
Sure, I have reverted this change.